README.md

```diff
@@ -57,11 +57,28 @@ Eglot provides a hook to format the buffer on saving:
    :hook
    (tuareg-mode . ocaml-eglot)
 -  (ocaml-eglot . eglot-ensure))
-+  (ocaml-eglot . eglot-ensure)
++  (ocaml-eglot . (lambda () 
++                   (add-hook #'before-save-hook #'eglot-format nil t))))
+```
+
+### Using `flycheck` instead of `flymake`
+
+Out of the box, eglot uses `Flymake` as a syntax checker. However, it
+is possible to use `flycheck`, via the
+[flycheck-eglot](https://github.com/flycheck/flycheck-eglot)
+package. If you have `flycheck-eglot` installed, you can change your
+configuration in this way:
+
+```diff
+ (use-package ocaml-eglot
```

Comment:
I"m guessing you can also add a `use-package` invocation for `flycheck-eglot`, so this example would be more complete.

Reply:
Fixed on https://github.com/tarides/ocaml-eglot/pull/37/commits/74f069941c6912f890a1be8c723ced7bac3e63fd